### PR TITLE
Readme added.

### PR DESCRIPTION
**Pull request by abdulrahimsulthan**
I created a README.md kindly verify the readme document and aprove it.

### CODE REVIEW

File: README.md
Comment (ostrich-white) [inline README.md:1]: Everything is good.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+2. The application will be running at http://localhost:3000.
+
+### In a Docker Container
+1. Build the Docker Image
+    ```sh
+    docker build -t asset-management-api .
+2. Run the Docker Container
+    ```sh
+    docker run -p 3000:3000 --name asset-management -d asset-management-api
+3. The application will be running at http://localhost:3000.
```
Comment (ostrich-white) [inline README.md:51]: try to add a new line here.

**Review by ostrich-white — APPROVED** [review]
Everything is good you can merge it.

File: README.md (hunk 1 above)
Comment (abdulrahimsulthan) [reply README.md:51]: I will do that in future.

**Comment by abdulrahimsulthan** [comment]
I merged the docs branch and delete it

**Pull request merged**